src/util/profile/prof_file.c

```diff
@@ -60,6 +59,10 @@ int profile_library_initializer(void)
 
     return k5_mutex_finish_init(&g_shared_trees_mutex);
 }
+
+#ifdef _WIN32
```

Comment:
This isn't quite the right fix.  Most of the time, `MAKE_FINI_FUNCTION` creates a static function - Windows is one of (I misread as "only") the cases where it doesn't.  I think it's more correct to just not warn about this function being potentially unused.  If it were more than one function doing this, I'd suggest modifying the definitions of `MAKE_FINI_FUNCTION`, but it's not.